### PR TITLE
Lay out the pie charts in a row

### LINKED ISSUE #619: Might be able to use pie charts on dahsboard for different types of errors
https://google-developers.appspot.com/chart/interactive/docs/gallery/piechart

Even better we could generate the charts in SVG without using JavaScript: http://www.webpagebytes.com/templates/template-SVG%20pie%20chart

### PR DESCRIPTION
**Pull request by elharo**
@suztomo fixes #619

### CODE REVIEW

File: dashboard/src/main/resources/templates/dashboard.ftl [hunk 1]
```diff
@@ -41,74 +41,76 @@
...
+      <th style="vertical-align:top">Dependency Convergence</th>
+      </tr>
+      <tr>
+        <td>${linkageErrorCount} out of ${totalArtifacts} artifacts 
+         ${plural(linkageErrorCount, "has", "have")} linkage errors.</td>
+        <td style="vertical-align:top">
+         ${localUpperBoundsErrorCount} out of ${totalArtifacts} artifacts 
+         ${plural(localUpperBoundsErrorCount, "does not", "do not")} pick the
+         latest versions of all artifacts in their own dependency tree.
+      </td>
```
Comment (suztomo) [inline dashboard/src/main/resources/templates/dashboard.ftl]: indent: add 2 spaces

File: dashboard/src/main/resources/templates/dashboard.ftl [hunk 2]
```diff
@@ -41,74 +41,76 @@
...
-      
-      <div>
-        <svg xmlns="http://www.w3.org/2000/svg" width="400" height="400">
+      <table>
+      <col width="${pieSize}" />
+      <col width="${pieSize}" />
+      <col width="${pieSize}" />
+      <col width="${pieSize}" />
+      <tr>
+      <th style="vertical-align:top">Linkage Errors</th>
```
Comment (suztomo) [inline dashboard/src/main/resources/templates/dashboard.ftl]: Would you move this style to CSS?

Comment (elharo) [reply dashboard/src/main/resources/templates/dashboard.ftl]: done

File: dashboard/src/main/resources/templates/dashboard.ftl (hunk 1 above)
Comment (elharo) [reply dashboard/src/main/resources/templates/dashboard.ftl]: done

**Pull request merged**